modules/codegen-v2/input/download.sh

```diff
@@ -12,6 +12,9 @@ curl -o video.xml https://raw.githubusercontent.com/KhronosGroup/Vulkan-Docs/ref
 echo Downloading OpenGL gl.xml
 curl -o gl.xml https://raw.githubusercontent.com/KhronosGroup/OpenGL-Registry/main/xml/gl.xml
 
+echo Downloading OpenXR xr.vml
```

Comment:
Use tag `release-1.1.49` instead of `master`: https://raw.githubusercontent.com/KhronosGroup/OpenXR-Docs/refs/tags/release-1.1.49/specification/registry/xr.xml